api/src/controllers/account.controller.js

```diff
@@ -2,8 +2,8 @@ import AccountService from '../services/account.service';
 
 
 const AccountController = {
-  createAnAccount(req, res) {
-    const createdAccount = AccountService.createAccount(req.userId, req.body.type);
+  async createAnAccount(req, res) {
+    const createdAccount = await AccountService.createAccount(req.userId, req.body.type);
     if (createdAccount.error) {
       return res.json({
         status: 401,
```

Comment:
you have errors without error messages. Please add descriptive error messages

Reply:
There are error messages. The error messages are coming from the services. That's how it is for the rest to

Reply:
I send the error messages as an object to the controller